fullscreen.bs

```diff
@@ -229,18 +232,18 @@ these steps:
    <code>TypeError</code> exception, and terminate these steps.
    <!-- cross-process; check is only needed on pending as it is recursive already -->
 
-   <li><p>Let <var>fullscreenElements</var> be an ordered set initially consisting of
+   <li><p>Let <var>fullscreenElements</var> be an <a>ordered set</a> initially consisting of
    <var>pending</var>.
 
    <li><p>While the first element in <var>fullscreenElements</var> is in a
-   <a>nested browsing context</a>, prepend its <a>browsing context container</a> to
+   <a>nested browsing context</a>, <a for=set>prepend</a> its <a>browsing context container</a> to
    <var>fullscreenElements</var>.
    <!-- cross-process -->
 
-   <li><p>Let <var>eventDocs</var> be an empty list.
+   <li><p>Let <var>eventDocs</var> be an empty <a>list</a>.
```

Comment:
Sure. I'm going to suggest removing it next, but let's not assume that's the final shape of the spec, more compat problems might revert it.